recipes/sail/all/conanfile.py

```diff
@@ -67,66 +58,78 @@ def requirements(self):
             self.requires("giflib/5.2.1")
         if self.options.with_jpeg2000:
             self.requires("jasper/2.0.33")
```

Comment:
Does Sail support Jasper 3.x? There is a new version available in Conan Center

Reply:
nope, it's still depends on 2.x